### ISSUE TITLE
Quest configuration - quartz ore

### ISSUE THREAD

**Issue by Meldiron** [JCulley/ProQuests-Documentation#251]
**Spigot account** - https://www.spigotmc.org/members/meldiron.92826/
<br>

**Plugin version** - v2.1.1.3.1
<br>

**Spigot Version** - This server is running Paper version git-Paper-486 (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)
<br>

**Link to log file** - not required - no errors, quest successfully loaded
<br>

**Additional information**
Quest config file:
```
# ======================================================
# 
# ProQuests Version: 2.1.1.3.1 build: 92826
# Created by: JC_Plays_MC
# 
# ======================================================

type: BREAK_QUEST
team: SOLO
duration: 0
repeat_delay: 60
repeatable: true
cancelable: true
command_start: true
command_join: true
check_all: false
cancel_after_timer: false
editable: false
disabled: false
join_active: false
require_invite: false
author: 38d6e62f-5f82-4d98-abb8-8b5ff42a4873
complete_actions:
- '[Title] &bQuest Finished;&7Reward:&f 50 €'
- '[ConsoleCommand] cmi money give %player% 50'
total: 64
allowed_materials:
- NETHER_QUARTZ_ORE;0
required_quests: []
blacklisted_quests:
- quartz_ore_2
allowed_worlds: []
blacklisted_worlds: []
allowed_regions: []
blacklisted_regions: []
complete_permissions: []
start_actions:
- '[Title] &bQuest Started;&7Goal: &fMine 64 quartz ore!'
return_actions: []
invite_actions: []
kick_actions: []
cancel_actions: []
required_materials: []
progress_actions: []
```

When I start the quest and start mining quartz ore, it doesnt count.
I have over 100 quests set up the same way, everything works except this one. For allowed materials I tried both NETHER_QUARTZ_ORE and QUARTZ_ORE. None of them gave any error (usually when I used wrong material name I got an error in console - material can not be null). This one doesnt work but doesnt give an error.

<br>

**Comment by JCulley** [comment]
Your log is always required. Regardless the quest uses Spigots material names so if you’re using the in-game editor to create the quests it would not allow the material if it does not exist. This means you’re more than likely using an invalid data value.

**Comment by Meldiron** [comment]
I dont use ingame editor, mostly it doesnt work for me (probably its just me doign wrogn stuff), I do it manually in config file.
Here is log file: https://pastebin.com/pLCyVzEE

Update: I have also got problem with terracotta. It would be awesome to have admin command like /quests debug  which will log everything I do so I just go and break specific block and copy material name from console.

For red terracotta I tried: TERRACOTA;14, RED_TERRACOTA;0, RED_TERRACOTTA;14, RED_HARDENED_CLAY;0, HARDENED_CLAY;14, RED_HARDENED_CLAY;14 - None of these works.

**Comment by JCulley** [comment]
Just tested and it worked fine on Spigot 1.13.2.

**Comment by Meldiron** [comment]
awesome solution.. works for me - means it has to work for everyone.

**Comment by JCulley** [comment]
I used the latest ProQuests version matching that of the one you’re using. I used the exact material you stated was not being recognized “NETHER_QUARTZ_ORE” the only difference is that I used an official Spigot build, which are the only versions I officially provide support for. If this is a problem caused by Paper Spigot then it’s not caused by the ProQuests so reach out to them, or use official Spigot builds. There is an absolutely 0% chance that your ProQuests version checks the material enums differently than the version I used so yes, it is a matter of if it worked for me it should work for everyone using the official Spigot builds. If you didnt want my analysis then don’t open an issue, it’s simple.
